.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: GitHub Community Support
+    url: https://github.com/orgs/community/discussions
```

Comment:
I’d link here to discourse ? @Hackerman342

Reply:
Yes!

Reply:
it has been added